rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -22,44 +22,38 @@
 import numpy as np
 import rocal_pybind as b
 import amd.rocal.types as types
+import ctypes
 
 class ROCALGenericImageIterator(object):
     def __init__(self, pipeline):
         self.loader = pipeline
-        self.w = b.getOutputWidth(self.loader._handle)
-        self.h = b.getOutputHeight(self.loader._handle)
-        self.n = b.getOutputImageCount(self.loader._handle)
-        color_format = b.getOutputColorFormat(self.loader._handle)
-        self.p = (1 if (color_format == int(types.GRAY)) else 3)
-        height = self.h*self.n
-        self.out_tensor = None
-        self.out_bbox = None
-        self.out_image = np.zeros((height, self.w, self.p), dtype = "uint8")
-        self.bs = pipeline._batch_size
+        self.batch_size = pipeline._batch_size
+        self.out = None
 
     def next(self):
         return self.__next__()
 
     def __next__(self):
+
         if(self.loader.isEmpty()):
             raise StopIteration
 
         if self.loader.run() != 0:
             raise StopIteration
 
-        self.loader.copyImage(self.out_image)
-        if((self.loader._name == "Caffe2ReaderDetection") or (self.loader._name == "CaffeReaderDetection")):
+        else:
+            self.output_tensor_list = self.loader.rocalGetOutputTensors()
 
-            for i in range(self.bs):
-                size = b.getImageNameLen(self.loader._handle,i)
-                print(size)
-                self.array = np.empty([1, size], dtype="<U15")
-                self.out=np.frombuffer(self.array, dtype=(self.array).dtype)
+        self.augmentation_count = len(self.output_tensor_list)
```

Comment:
Augmentation count is not used

Reply:
Removing it for now. WIll need it for outputs different augmentations with different shapes and dims